prior_localization/run_scripts/run_ephys_decoding.py

```diff
@@ -18,19 +18,21 @@
 n_pseudo = int(args.n_pseudo)
 n_per_job = int(args.n_per_job)
 output_dir = str(args.output_dir)
-target = str(args.output_dir)
-
-# TODO: settings for the particular target
+target = str(args.target)
 
 # Get session idx
 session_idx = int(np.ceil(job_idx / (n_pseudo / n_per_job)) - 1)
 
-# Set of pseudo sessions for one session, first session is always the real one, indicated by -1
+# Set of pseudo sessions for one session
 all_pseudo = list(range(n_pseudo))
-all_pseudo[0] = -1
 # Select relevant pseudo sessions for this job
 pseudo_idx = int((job_idx - 1) % (n_pseudo / n_per_job) * n_per_job)
 pseudo_ids = all_pseudo[pseudo_idx:pseudo_idx+n_per_job]
+# Shift by 1; old array starts at 0, pseudo ids should start at 1
```

Comment:
Are you sure? I thought I tested this quite extensively. Mind that the SLURM_ARRAY_TASK_ID (passed into job_idx) is 1-based, not 0-based